src/app/stJude/StJudeToaster.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useState, useEffect } from 'react';
+import classNames from 'classnames';
+import { AppIcon, disabledIcon } from '../shell/icons';
+import './st-jude-toaster.scss';
+import stJudeImage from '../../images/stjude_finding-cures_londyn-learn-more_animated_300x50.gif';
+
+const StJudeToaster = () => {
+  const [isMinimized, setIsMinimized] = useState(true);
+  const [visible, setVisible] = useState(false);
+
+  useEffect(() => {});
+
+  const openCampaignWindow = () => {
+    const campaign = window.open(
+      'https://tiltify.com/@thisisdim/dim-2019-fundraising-event/donate',
+      '_blank'
+    );
+
+    if (campaign !== null) {
+      campaign.opener = null;
+    }
+  };
+
+  const openCampaign = () => {
+    setIsMinimized(false);
+    setTimeout(() => {
+      setVisible(true);
+    });
+  };
+
+  const closeCampaign = () => {
+    setIsMinimized(true);
+    setVisible(false);
+  };
+
+  return (
+    <>
+      <div
+        className={classNames(`st-jude-toaster`, {
+          minimized: isMinimized,
+          maximized: !isMinimized
+        })}
+      >
+        {!isMinimized && (
```

Comment:
You can write this as a ternary, like `isMinimized ? (some content) : (other content)`.